prisma/schema.prisma

```diff
@@ -54,8 +55,8 @@ model Prescription {
   Patient     Patient?  @relation(fields: [patientId], references: [id])
   patientId   Int
   balance     Float
-  location    Location  @relation(fields: [locationId], references: [id])
-  lockerBox   LockerBox @relation(fields: [lockerBoxId], references: [id])
```

Comment:
I might be wrong but I think prisma docs suggest model fields are kept lowercased